create.py

```diff
@@ -16,9 +16,9 @@
 
 
 
-cats = [Category("News", "news"), Category("Sports", "sports"),
-        Category("Weather", "weather"),
-        Category("Woove", "woove")]
+cats = [Category("News", "news", True), Category("Sports", "sports", True),
```

Comment:
If all of these are True, why not set that to be the default so no change is needed here? See comment below...